Ocean/cube4km.geojson

```diff
@@ -0,0 +1,155 @@
+{
+    "type": "Feature",
+    "properties": {
+        "data_id": "ocean-1M-4km-64x256x256-1.0.0.zarr",
+        "title": "Ocean Data Cube 4km v1.0.0",
+        "description": "Ocean Data Cube for Particulate Organic Carbon (POC)",
+        "version": "1.0.0",
+        "recipe": "",
+        "changes": "",
+        "license": "Terms and conditions of the DeepESDL data distribution",
+        "time_range": ["1997-09-01","2020-03-01"],
+        "time_period": "1M",
+        "spatial_ref": "EPSG:4326",
+        "spatial_bbox": [-180, -90, 180, 90],
+        "spatial_res": [0.04166412353515625, 0.04166412353515625],
+        "variables": [
+            {
+                "name": "POC",
+                "long_name": "Particulate Organic Carbon",
+                "dtype": "float32",
+                "dims": ["time","y","x"],
+                "units": "mg m^-3",
+                "fill_value": "nan",
+                "valid_min": null,
```

Comment:
https://github.com/deepesdl/dataset-spec/tree/main/dataset-spec#data-variable---general

> If valid_min, valid_max are given, their values should be numbers, not null. Same applies to the value items of valid_range.

Please apply this for all occurences.

Reply:
I decided to delete these attributes also as they are optional.